src/routes/movies.py

```
@@ -1,13 +1,241 @@
-from fastapi import APIRouter, Depends, HTTPException, Query
-from sqlalchemy import select, func
-from sqlalchemy.exc import IntegrityError
+from __future__ import annotations
+
+from math import ceil
+from typing import Optional
+from urllib.parse import urlparse
+
+from fastapi import APIRouter, Depends, HTTPException, Query, Request, status
+from sqlalchemy import and_, func, select
 from sqlalchemy.ext.asyncio import AsyncSession
-from sqlalchemy.orm import joinedload
 
-from database import get_db, MovieModel
-from database.models import CountryModel, GenreModel, ActorModel, LanguageModel
+from ..database.models import (
+    MovieModel,
+    GenreModel,
+    ActorModel,
+    CountryModel,
+    LanguageModel,
+)
+from ..database.session_sqlite import get_db
+from ..schemas.movies import (
```

You're re-importing FastAPI/SQLAlchemy names here which duplicates earlier imports. Having duplicate/conflicting imports makes the module harder to reason about and may hide errors. Clean up imports so every name is imported exactly once and at the top of the file.